io.openems.edge.evcs.api/src/io/openems/edge/evcs/api/Evcs.java

```diff
@@ -411,7 +411,7 @@ public default void _setPhases(Phases value) {
 	 * @param value the next value
 	 */
 	public default void _setPhases(Integer value) {
-		if (value == null) {
+		if (value == null || value == 0) {
```

Comment:
@sebastianasen: Ist das semantisch richtig? Wenn ich nur diesen Code lese ist es für mich nicht intuitiv. Ich würde das `if` am liebsten sogar komplett weglassen und in diesem Fall unten die `IllegalArgumentException` werfen. Ist das `null`/`0`-Handling wirklich notwendig?

Reply:
Die Exception müsste ausreichen.
Wäre aber eventuell auch etwas für einen separaten PR.

Reply:
Ich habe mir alle Fälle angeschaut, in denen diese Methode aufgerufen wird. Es ist tatsächlich eine kleine Verbesserung, hier auch auf `0` abzuprüfen. So wie es aktuell ist, gefällt es mir aus mehreren Gründen nicht:
1. Es gäbe eigentlich bereits ein Enum, das man einfach als Parameter nutzen könnte
2. Dass `null` bzw. `0` dreiphasig bedeutet ist nicht intuitiv. Ich habe verstanden, dass das im UI als Fallback benötigt wird, aber dann sollte man es dort lösen
3. Die Methode wirft eine RuntimeException, was wir eigentlich im Code vermeiden, Konkret würden hier in dem Fall bei den aufrufenden Methoden einige wichtige Schritte nicht mehr bearbeitet werden.

Das sind aber alles Themen für einen separaten PR und unabhängig von der Webasto Unite.